### PR TITLE
#33 create a nextflow module for virsorter2

### LINKED ISSUE #33: Create a Nextflow module for Virsorter2
### Description of feature

Virsorter2 is not available through nf-core. Write a nf-core standard [VirSorter2](https://github.com/jiarong/VirSorter2) module

- [x]  include a Singularity/docker container
- [x]  include a bioconda container

### PR DESCRIPTION
**Pull request by jimmyliu1326**
- Created a local module for Virsorter2
- Part of the phage prediction subworkflow
- Closes #33

### CODE REVIEW

File: modules/local/virsorter2/main.nf [hunk 1]
```diff
@@ -0,0 +1,38 @@
+process VIRSORTER2 {
+    tag "$meta.id"
+    label 'process_medium'
+    conda "bioconda::virsorter=2.2.3-pyhdfd78af_1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://jiarong/virsorter:2.2.3' :
+        'jiarong/virsorter:2.2.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
```
Comment (anwarMZ) [inline modules/local/virsorter2/main.nf:10]: take db from virsorter/setup module

File: modules/local/virsorter2/main.nf [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    path "${prefix}/final-viral-combined.fa"  , emit: fasta
+    path "${prefix}/final-viral-score.tsv"    , emit: score
+    path "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args     = task.ext.args ?: ''
+    def db       = params.virsorter_db ?: '/db'
```
Comment (anwarMZ) [inline modules/local/virsorter2/main.nf:23]: check if input ch_db==null then db = /db
else: db==ch_db

Comment (jimmyliu1326) [reply modules/local/virsorter2/main.nf]: resolved in https://github.com/cidgoh/nf-seqqc/pull/52/commits/30067fb3b5f244ca93b80296dce93214cc55c1b6

File: modules/local/virsorter2/main.nf (hunk 1 above)
Comment (jimmyliu1326) [reply modules/local/virsorter2/main.nf]: resolved in https://github.com/cidgoh/nf-seqqc/commit/30067fb3b5f244ca93b80296dce93214cc55c1b6

**Pull request merged**